### ISSUE TITLE
deploy:version replacs too much

### ISSUE THREAD

**Issue by DanielRuf** [zurb/foundation-sites#11152]
https://github.com/zurb/foundation-sites/blob/c4602f01b475b1775cd75a55b39b86a2f7b264ba/gulp/config.js#L70 would also affect all normal dependencies like it did in the past.

**Comment by ncoden** [comment]
True. I think the only same approach would be to replace the version manually.
